### PR TITLE
[4.0] Improve layout of transitions overview

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Improve the transitons view by adding the icon for the respective condition to every stage.

After  Patch
transactions_after

### Testing Instructions
Compare the view transition before and after patch.

### Actual result
transactions_before

### Documentation Changes Required
yes, layout has changed

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_workflow.ini [hunk 1]
```diff
@@ -90,6 +90,10 @@ COM_WORKFLOW_TO_STAGE_DESC="Select target stage."
 COM_WORKFLOW_TRANSITION="Transition"
 COM_WORKFLOW_TRANSITIONS="Transitions"
 COM_WORKFLOW_TRANSITIONS_LIST="%s Transitions List"
+COM_WORKFLOW_TRANSITIONS_CONDITION_ARCHIVED="Condition is archived"
+COM_WORKFLOW_TRANSITIONS_CONDITION_PUBLISHED="Condition is published"
+COM_WORKFLOW_TRANSITIONS_SCONDITION_UNPUBLISHED="Condition is unpublished"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_workflow.ini]: typo SCON....

?

Comment (Quy) [inline administrator/language/en-GB/en-GB.com_workflow.ini]: typo `SCONDITION`

Comment (Quy) [reply administrator/language/en-GB/en-GB.com_workflow.ini]: sort in alpha order

Comment (chmst) [reply administrator/language/en-GB/en-GB.com_workflow.ini]: oops

**Comment by brianteeman** [comment]
I know it is shorter but it doesn't make sense to me and would be better as

The condition of items at this stage is published 

So you can perhaps reuse existing string

COM_WORKFLOW_CONDITION . JPUBLISHED

**Comment by chmst** [comment]
Thank you @brianteeman , good idea.

**Review by brianteeman — APPROVED** [review]

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 3de041e27864e96edeea9d90e724c1c6777a91d6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22315">issues.joomla.org/tracker/joomla-cms/22315</a>.</sub>

**Comment by bembelimen** [comment]
I have tested this item :white_check_mark: successfully on 3de041e27864e96edeea9d90e724c1c6777a91d6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22315">issues.joomla.org/tracker/joomla-cms/22315</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

File: administrator/components/com_workflow/tmpl/transitions/default.php [hunk 2]
```diff
@@ -124,18 +125,50 @@
...
+											$condition = TEXT::_('JTRASHED');
+										elseif ($item->to_condition == Workflow::CONDITION_PUBLISHED):
+											$icon = 'icon-publish';
+											$condition = TEXT::_('JPUBLISHED');
+										elseif ($item->to_condition == Workflow::CONDITION_UNPUBLISHED):
+											$icon = 'icon-unpublish';
+											$condition = TEXT::_('JUNPUBLISHED');
+										endif; ?>
+										<span class="<?php echo $icon; ?>" aria-hidden="true"></span>
+										<span class="sr-only"><?php echo TEXT::_('COM_WORKFLOW_CONDITION') . $condition; ?></span>
```
Comment (Quy) [inline administrator/components/com_workflow/tmpl/transitions/default.php:169]: Should this be same as #22326 without the `sr-only` text?

**Comment by wilsonge** [comment]
Can you fix conflicts and address @Quy 's question please :)

File: administrator/components/com_workflow/tmpl/transitions/default.php (hunk 2 above)
Comment (chmst) [reply administrator/components/com_workflow/tmpl/transitions/default.php:169]: No,it is not the same. In transitions, I have only numbers as conditions and need to map it via Constants. In stages, the Condition is already a text.

**Comment by wojsmol** [comment]
@chmst Please revert https://github.com/joomla/joomla-cms/pull/22315/commits/1016abd5baf5e6f471ffb9752b675898bdfb746d and only fix merge conflicts in administrator/components/com_workflow/tmpl/transitions/default.php file. In https://github.com/joomla/joomla-cms/pull/22315/commits/1016abd5baf5e6f471ffb9752b675898bdfb746d you merged outdated 4.0-dev state into this branch.

**Comment by wojsmol** [comment]
@franz-wohlkoenig Please remove RTC label temporally - see my previous comment.

**Comment by chmst** [comment]
sorry, I cannot revert the commot need help someone for help. I do it asap

**Comment by chmst** [comment]
@wojsmol I have pulled the latest v4.0 dev branch of joomla. Is this corret now?

**Comment by wojsmol** [comment]
No, you still have 50 files changed. This should fix the issue:
```
git checkout -b workflow-transitions-layout-backup #create local backup branch and switch to it.
git checkout workflow-transitions-layout # switch back to This PR branch.
git reset --hard 3de041e27864e96edeea9d90e724c1c6777a91d6 # reset PR to good state.
git push <your-remote-name> -f # push changes to github - replace <your-remote-name> with your joomla fork remote name. This can be checked with git remote -vv 
```
Then we are at `Fix quotes, Use existing language strings` commit and then only conflicts neds to to be fixed.

**Comment by chmst** [comment]
Could I delete this branch? It is simpler to make a new one - it is not much code.

**Comment by wojsmol** [comment]
You can - this will close this PR, If you create new branch with the same name then you should have reopen option on bottom of comment form on github..

**Comment by ghost** [comment]
Closed as stated in #22343 by @chmst.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22315">issues.joomla.org/tracker/joomla-cms/22315</a>.</sub>

**Pull request closed without merging**